dtfs-central-api/src/helpers/execute-with-sql-transaction.ts

```diff
@@ -71,6 +71,8 @@ export const executeWithSqlTransaction = async <ReturnValue>(functionToExecute:
 
     return result;
   } catch (error) {
+    console.error('Error thrown within SQL transaction: %o', error);
```

Comment:
Thoughts on adding a `console.info` at the top as well?

Reply:
I don't think that would add much value